Source/SwiftLintCore/Models/Issue.swift

```diff
@@ -20,6 +20,21 @@ public enum Issue: LocalizedError, Equatable {
     /// Used rule IDs are invalid.
     case invalidRuleIDs(Set<String>)
 
+    /// Found a rule configuration for a rule that is not present in only_rules.
+    case ruleNotPresentInOnlyRules(ruleID: String)
```

Comment:
The issues are about the configuration, aren't they?
```suggestion
    case configurationForRuleNotPresentInOnlyRules(ruleID: String)
```